backend/src/main/java/hanglog/expense/domain/CategoryExpense.java

```diff
@@ -0,0 +1,34 @@
+package hanglog.expense.domain;
+
+import hanglog.category.domain.Category;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import lombok.Getter;
+
+@Getter
+public class CategoryExpense {
+
+    private static final int PERCENTAGE_CONSTANT = 100;
+
+    private final Category category;
+    private final int amount;
+    private final BigDecimal percentage;
+
+    public CategoryExpense(
+            final Category category,
+            final int amount,
+            final int totalAmount
+    ) {
```

Comment:
여기는 한 줄에 놔도 길이 제한 안넘어갈 것 같은데요?!
```suggestion
    public CategoryExpense(final Category category, final int amount, final int totalAmount) {
```

저희 컨벤션
- 매개변수 4개부터 세로로 작성
- 단, 매개변수 3개여도 가로 라인(120)을 넘으면 세로로 작성